src/pybind/mgr/dashboard/frontend/src/app/shared/models/topic.model.ts

```diff
@@ -0,0 +1,57 @@
+interface Destination {
+  push_endpoint: string;
+  push_endpoint_args: string;
+  push_endpoint_topic: string;
+  stored_secret: string;
+  persistent: boolean;
+  persistent_queue: string;
+  time_to_live: number;
+  max_retries: number;
+  retry_sleep_duration: number;
+}
+
+export interface TopicModel {
```

Comment:
tbh, its clear from the file name that this is a modal, we don't need to repeat that for every interface.
```suggestion
export interface Topic {
```

Reply:
Suggested changes are done